docs/source/user-guide/crate-configuration.md

```diff
@@ -25,7 +25,47 @@ control DataFusion's behavior.
 
 [configuration settings]: configs.md
 
-## Add latest non published DataFusion dependency
+## Crate features
+
+The `datafusion` crate has several [features] which can be specified in your `Cargo.toml`. For example
+to enable support for `avro` you can put the following in your `Cargo.toml`:
+
+```toml
+datafusion = { version = "45", features = ["avro"] }
+```
+
+[features]: https://doc.rust-lang.org/cargo/reference/features.html
+
+You can always check the [Cargo.toml] file for the most up to date list of features.
+
+[cargo.toml]: https://github.com/apache/datafusion/blob/main/datafusion/core/Cargo.toml#L39
+
+### Default features:
+
+- `nested_expressions`: functions for working with nested type function such as `array_to_string`
+- `compression`: reading files compressed with `xz2`, `bzip2`, `flate2`, and `zstd`
+- `crypto_expressions`: cryptographic functions such as `md5` and `sha256`
+- `datetime_expressions`: date and time functions such as `to_timestamp`
+- `encoding_expressions`: `encode` and `decode` functions
+- `math_expressions`: `sqrt` and other math functions
```

Comment:
This doesn't appear to be in the defaults list here:
https://github.com/apache/datafusion/blob/40bb75fdbb9c19344bcd3953b5d26e672f2ae99a/datafusion/core/Cargo.toml#L49

Reply:
That was an excellent call. I fixed it in df190ada7

However, I think since the subcrates already have their own default features, it is likely not possible to remove these various functon packages 🤔 maybe we should fix that in a following issue